### ISSUE TITLE
Add scenario tags in the scenario selector

### ISSUE THREAD

**Issue by rohan011001** [Avaiga/taipy#1235]
**Description**
having scenario tags displayed in the scenaro slector help spotting a specific scenario. Can be hard.
Each tag value must have one color. Like github labels
![image](https://github.com/Avaiga/taipy/assets/168116259/b29c6694-b639-4156-a3bc-2d558abbbca8)
In particular when they are defined as authorized_tags as specified in the [documentaion](https://docs.taipy.io/en/release-2.4/manuals/core/entities/scenario-cycle-mgt/#tag-or-untag-a-scenario).

A similar improvement also for generic labels and not only tags. If scenario has label key in the property dictionary with a set of strings as value, the selector will display.

For tags we should provide a noolean to activate the property
```
import taipy.gui.builder as tgb
 
tgb.scenario_selector("{scenario}", tags=True)
```

For generic labels we should provide the dict key

```
import taipy.gui.builder as tgb
 
tgb.scenario_selector("{scenario}", label=label)
```

I can help on python only. Please.

**Acceptance Criteria**
- [ ] Ensure new code is unit tested, and check code coverage is at least 90%
- [ ] Propagate any change on the demos and run all of them to ensure there is no breaking change
- [ ] Ensure any change is well documented

**Comment by FlorianJacta** [comment]
I like this idea. It makes the scenario selector a lot more practical and with a better UX. We should find a way to design it

**Comment by jrobinAV** [comment]
Hello @rohan011001 

Thank you for your proposal.

Can you have a look to the issue #1075. This is somehow related to improve user experience and to make it easier to select one scenario in a long list.
Do you believe it can solve your issue as well?

**Comment by rohan011001** [comment]
@jrobinAV 
Yes I think filter and sort is better than display tags. Not exclusive. If I have filter, I don't need tags.

But can you show me the documentation? I can't find it. How can I use filters and sort?

**Comment by FredLL-Avaiga** [comment]
filter is coming in the next version and will be documented then
Would you like to close this issue @rohan011001  ?

**Comment by jrobinAV** [comment]
We believe the use case makes sense. We have to discuss the possible solutions.

**Comment by mohammed-anwar404** [comment]
hey there, I would like to pick this as my first issue to contribute to Hacktoberfest. I am completely new to open source contributions. Please guide me along the way.

**Comment by FlorianJacta** [comment]
Great, let me assign this to you @mohammed-anwar404. You can ask @jrobinAV and @FredLL-Avaiga for more information.

Here is [documentation](https://github.com/Avaiga/taipy/blob/develop/CONTRIBUTING.md) on contributing to Taipy.

Thank you for your contribution!

**Comment by mohammed-anwar404** [comment]
Thanks, @FlorianJacta for assigning me this issue. I am looking forward to learn and to contribute. 
If I get stuck I will reach out to the Taipy team for help.

**Comment by Harish-2003** [comment]
So are u saying that we need to add the colour to the scenarios tags?

**Comment by FlorianJacta** [comment]
To add these tags inside the scenario selector like GitHub does for labels. For example, in this app, we have a scenario selector and it does not contain the tags that scenarios have. https://supply-chain.taipy.cloud/scenario

**Comment by jrobinAV** [comment]
Hello @mohammed-anwar404, Are you working on this issue?

**Comment by mohammed-anwar404** [comment]
Hello @jrobinAV, Currently, I am not working on this issue. You can assign this issue to anyone who likes to take it up. I do need some time to understand and contribute.

**Comment by jrobinAV** [comment]
OK, Thank you.
